html/feed/feed.js

```diff
@@ -339,4 +340,121 @@ container.appendChild (card);
 });
 
 
-}
\ No newline at end of file
+}
+
+
+
+
+const filterBtnTag = document.querySelector ("#tag")
+
+
+async function getTags (headerOptions){
+  try {
+      const response = await fetch (`${baseUrl}/auction/listings?_tag=my_tag&_active=true`,headerOptions); 
+      const tag= await response.json();
+      console.log (tag)
+      return tag;
+  } catch (error){
+      console.log (error);
+  }
+}
```

Comment:
Why are we fetching from the auction house api? we are working with the social api. this is not our assignment